### PR TITLE
feat: chained printing method

### PR DESCRIPTION
**Pull request by zchryst**
Hey @tr3v3r - made a bit of progress today towards the chained printing method, looking for any feedback on this draft. I'll be looking at the image tomorrow

### CODE REVIEW

**Comment by tr3v3r** [comment]
@zchryst Grate job! Thanks a lot! 

Are you planning to add the same feature for Android?

p.s. I'll review today / tomorrow the draft

**Comment by zchryst** [comment]
@tr3v3r Sorry, I'm afraid our project is only using iOS and due to time constraints, I won't be able to commit to the android side. 

I'm hoping that this can at least lay the groundwork for someone else to do that part?

**Comment by zchryst** [comment]
Alrighty - I've got the image coming through as well, now converting to a proper PR 🥳

File: src/printing.ts [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+  /**
+   * Send the current array of commands to the printer
+   *
+   * @param  {string}   value  String to encode
+   * @return {object}          Encoded string as a ArrayBuffer
+   *
+   */
+  _send(value: any) {
+    return EscPosPrinter.printBuffer(value);
```
Comment (tr3v3r) [inline src/printing.ts:58]: Could please also add the stub method for Android to not cause a crash for Android users?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 We want this community to be friendly and respectful to each other. Please follow it in all your interactions with the project.
 
+## Before cloning this repository
+
+This repository relies on [git large file storage](https://github.com/git-lfs/git-lfs)
+
+Ensure that you have installed git-lfs prior to cloning the repository.
+
 ## Development workflow
```
Comment (tr3v3r) [inline CONTRIBUTING.md:11]: It also would be nice if you could add short docs or examples in the README.md file

File: src/printing.ts (hunk 1 above)
Comment (tr3v3r) [inline src/printing.ts:58]: Could you please add Promise wrapper like for main methods (see getPaperWidth at index.ts ) to provide correct completion for printing task?

**Comment by tr3v3r** [comment]
@zchryst Thanks. I've left several comments. Could you please take a look and then we can merge it into the main branch?


I'll provide the same functionality for Android as well on my own.

**Review by tr3v3r — CHANGES_REQUESTED** [review]

**Comment by zchryst** [comment]
Thanks @tr3v3r - I'll try fix that up towards the end of the week

**Comment by zchryst** [comment]
@tr3v3r - apologies for the delay, I've added the stub, updated the docs, and added the promise wrapper - seems to all be working rather well, though I do not have an android device to test with I'm afraid

**Review by tr3v3r — APPROVED** [review]

**Comment by tr3v3r** [comment]
merged. I'll make a short check and will release a new version soon

**Comment by tr3v3r** [comment]
@zchryst hi! I've just tested your solution regarding printing image and it seems that it doesn't work ;( ( I'm tested on my Epson printer TM-T20II ). For some reason, the base64 image is not processed correctly. 

On the other hand, I'll found another solution to put images into iOS/Android assets folder ( on native side ) and print image from the giving source just using asset name like:

- ios 
   - assets 
        - image.png
        
  ```
  
  .image('image', 200)
  
  ```
  
  So for now I'll keep your method in the code and rename it to `imageFromBase64` to use in chain printing and add another method 'imageFromAssets' to print from source I've mentioned above.
  
  It would be nice if could investigate what the problem and fix if it possible. Thanks

**Pull request merged**